screens/History.tsx

```diff
@@ -69,60 +73,128 @@ export default function History({ navigation, route }) {
 
       <View style={styles.main}>
         <Text style={styles.name}>{"History"}</Text>
-		<Text style={styles.message}>Coming soon </Text>
-
-       
+        <View style={styles.order}>
+          <View style={styles.details}>
+            <ELoan />
+            <View style={styles.title}>
+              <Text
+                style={{
+                  color: "#074A74",
+                  fontFamily: "Montserrat_700Bold",
+                }}
+              >
+                E-LOAN ₦60,000{" "}
+              </Text>
+              <Text style={{ color: "#000", fontSize: 12 }}>
+                Order ID: 36584AG6
+              </Text>
+            </View>
+          </View>
+          <Text style={{ color: "#000", fontSize: 13 }}>12/22/2022</Text>
```

Comment:
why are we hard coding this?

Reply:
Just a place holder. It will be my task for today